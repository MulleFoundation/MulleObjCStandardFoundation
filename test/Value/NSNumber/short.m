#ifndef __MULLE_OBJC__
# import <Foundation/Foundation.h>
#else
# import <MulleObjCStandardFoundation/MulleObjCStandardFoundation.h>
#endif


main()
{
   NSNumber   *value;

   value = [NSNumber numberWithShort:0];
   printf( "%d\n" , [value intValue]);


   value = [NSNumber numberWithShort:-1848];
   printf( "%d\n" , [value intValue]);

   value = [NSNumber numberWithShort:+1848];
   printf( "%d\n" , [value intValue]);

   return( 0);
}
