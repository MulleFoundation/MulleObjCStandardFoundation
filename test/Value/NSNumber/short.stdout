0
-1848
1848
